.github/CODEOWNERS

```diff
@@ -702,7 +702,7 @@ packages/analytics/shippers/elastic_v3/common @elastic/kibana-core
 packages/analytics/shippers/elastic_v3/server @elastic/kibana-core
 packages/analytics/shippers/fullstory @elastic/kibana-core
 packages/analytics/shippers/gainsight @elastic/kibana-core
-packages/content-management/table_list @elastic/kibana-global-experience
+packages/content-management/table_list @elastic/shared-ux
```

Comment:
Did you mean to have these CODEOWNERS changes?

Reply:
This was probably an outdated branch, updated and it went away